doc/src/site/sphinx/tutorials/sw_google_cloud_dataproc.rst

```diff
@@ -0,0 +1,184 @@
+Running Sparkling Water on Google Cloud Dataproc
+------------------------------------------------
+
+This section describes how to run Sparkling Water on `Google Cloud Dataproc <https://cloud.google.com/dataproc/docs/concepts/overview>`__. 
+
+Perform the following steps to start Sparkling Water ``H2OContext`` on Cloud Dataproc. 
+
+**Note**: Be sure to allocate enough resources handle your data. As a minimum recommendation, if your data is 1GB, then you would want 5GB total (for example).
```

Comment:
should be `to handle your data` ? Also I think the rule of thumb is 4 times memory of the size of your data in H2O